python_modules/libraries/dagster-k8s/dagster_k8s_tests/unit_tests/test_executor.py

```diff
@@ -645,11 +649,20 @@ def test_step_raw_k8s_config_inheritance(
     )
 
     # Verifies that raw k8s config for step pods is pulled from the container context and
-    # dagster-k8s/config tags on the op, but *not* from tags on the job
+    # executor-level config and dagster-k8s/config tags on the op, but *not* from tags on the job
     executor = _get_executor(
         k8s_run_launcher_instance,
         reconstructable(bar_with_tags_in_job_and_op),
-        {},
+        {
+            "step_k8s_config": {  # injected into every step
+                "container_config": {
+                    "working_dir": "MY_WORKING_DIR",  # set on every step
+                    "volume_mounts": (
+                        OTHER_VOLUME_MOUNTS_TAGS
+                    ),  # overridden at the op level, so ignored
```

Comment:
confused by this "ignored" comment but also the value aligns with the asserted value below - should this ignored thing be different?

Reply:
oops, this was supposed to be THIRD_VOLUME_MOUNTS_TAGS